aten/src/ATen/cpu/vec/vec_half.h

```diff
@@ -0,0 +1,40 @@
+#include <ATen/cpu/vec/intrinsics.h>
```

Comment:
is this part of code from Cao E's patch ?

Reply:
Yes, and I have removed the Cao E's patch since the PR has landed.